### PR TITLE
Require the Heroku app name for bin/setup

### PR DESCRIPTION
**Pull request by mxie**
Users should be joining their own deployed Heroku app.

### CODE REVIEW

File: bin/setup [hunk 1]
```diff
@@ -6,36 +6,41 @@
 # Exit if any subcommand fails
 set -e
 
-# Set up Ruby dependencies via Bundler
-gem list bundler --installed > /dev/null || gem install bundler
-bundle install
+if [ -z "$1" ]; then
```
Comment (drapergeek) [inline bin/setup]: This seems like a lot of work to require each time. How about instead we add a line that recommends that the user add their own. If they want to remove that line, they're welcome to do so and stop being bugged about it.

Comment (mxie) [inline bin/setup]: You're right. That seems easiest. Done!

**Comment by mxie** [comment]
Ready for re-review.

**Comment by drapergeek** [comment]
:+1:

**Pull request merged**